test/framework/fileConversion.go

```diff
@@ -166,8 +166,10 @@ func copyIfNotPresent(src, tgtDir string) (string, error) {
 	if err != nil && !os.IsNotExist(err) {
 		return "", errors.Wrap(err, "Unexpected error stating file")
 	}
-	if err = doCmd("cp", src, tgtDir); err != nil {
-		return "", err
+	if os.IsNotExist(err) {
+		if err = doCmd("cp", src, ret); err != nil {
```

Comment:
nit: could be combined with `&&` but no big deal...

Reply:
Other than the code not quite matching the comments, this is just personal preference. I'd do:
```
// If no copy is performed, the original src string is returned.   // <==== true?
func copyIfNotPresent(src, tgtDir string) (string, error) {
	ret := filepath.Join(tgtDir, filepath.Base(src))
	_, err := os.Stat(ret)
	if err == nil {
                return src, nil
        }
	if !os.IsNotExist(err) {
                return src, errors.Wrap(err, "Unexpected error stating file")
        }
        if err = doCmd("cp", src, ret); err != nil {
		return src, err
	}
	return ret, nil
}
```